### PR TITLE
Add note about additional config for ClassicCompute VMs

### PR DESCRIPTION
**Pull request by andreiztm**

### CODE REVIEW

**Comment by illfated** [comment]
#sign-off

**Comment by andreiztm** [comment]
Hi, could someone please review and merge this? Thanks, Andrei.

**Comment by illfated** [comment]
@eross-msft : Would you be available to follow up on this PR, or do we need to check with someone else?

File: WindowsServerDocs/get-started/extended-security-updates.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ For more information, see the [Extended Security Updates frequently asked questi
 
 If you run Windows Server 2008 or 2008 R2 VMs in Azure, they're automatically enabled for Extended Security Updates. You don't need to configure anything, and there's no additional charge for using Extended Security Updates with Azure VMs. Extended Security Updates are automatically delivered to Azure VMs if they're configured to receive updates.
 
+> [!NOTE]
+> Microsoft.ClassicCompute type VMs require additional configuration for ESU deployment, as they do not have access to the Azure Resource Manager, in particular the [Azure Instance Metadata Service](https://docs.microsoft.com/azure/virtual-machines/windows/instance-metadata-service), which is used for evaluating ESU eligibility. Please contact [Microsoft support](http://support.microsoft.com/contactus?PID=17336) for more help.
```
Comment (eross-msft) [inline WindowsServerDocs/get-started/extended-security-updates.md:45]: @andreiztm  Please change the following: 
- ESU to Extended Security Updates. For some reason, even though we show the acronym, it's always spelled out completely in the text.
- "Do not have" to "don't have"

I'm unclear whether your are a Microsoft employee and subject matter expert on this, or if we need to have someone review this text before publishing. Can you clarify your role? Thanks!

**Comment by andreiztm** [comment]
@eross-msft thank you for the suggestions, I added them in (hopefully correct) and sent you an e-mail confirming my role.

File: WindowsServerDocs/get-started/extended-security-updates.md [hunk 2]
```diff
@@ -41,6 +41,9 @@ For more information, see the [Extended Security Updates frequently asked questi
 
 If you run Windows Server 2008 or 2008 R2 VMs in Azure, they're automatically enabled for Extended Security Updates. You don't need to configure anything, and there's no additional charge for using Extended Security Updates with Azure VMs. Extended Security Updates are automatically delivered to Azure VMs if they're configured to receive updates.
 
+> [!NOTE]
+> Microsoft.ClassicCompute type VMs require additional configuration for Extended Security Updates deployment, as they don't have access to the Azure Resource Manager, in particular the [Azure Instance Metadata Service](https://docs.microsoft.com/azure/virtual-machines/windows/instance-metadata-service), which is used for determining ESU eligibility. Please contact [Microsoft support](http://support.microsoft.com/contactus?PID=17336) for more help.
```
Comment (eross-msft) [inline WindowsServerDocs/get-started/extended-security-updates.md:45]: @andreiztm  After looking at this a bit more, when we spell out ESU it gets a bit long. I would suggest chopping it down slightly. What do you think about: 
"Microsoft.ClassicCompute VMs require additional configuration for Extended Security Updates deployment, as they don't have access to the [Azure Instance Metadata Service](https://docs.microsoft.com/azure/virtual-machines/windows/instance-metadata-service), which determines your Extended Security Updates eligibility. Please contact [Microsoft support](http://support.microsoft.com/contactus?PID=17336) for more help."

**Comment by andreiztm** [comment]
@eross-msft thank you very much for the ongoing feedback, learning a lot. I added the latest changes.

**Review by illfated — APPROVED** [review]
Looks very good to me.

**Comment by illfated** [comment]
#sign-off

File: WindowsServerDocs/get-started/extended-security-updates.md [hunk 3]
```diff
@@ -41,6 +41,9 @@ For more information, see the [Extended Security Updates frequently asked questi
 
 If you run Windows Server 2008 or 2008 R2 VMs in Azure, they're automatically enabled for Extended Security Updates. You don't need to configure anything, and there's no additional charge for using Extended Security Updates with Azure VMs. Extended Security Updates are automatically delivered to Azure VMs if they're configured to receive updates.
 
+> [!NOTE]
+> Microsoft.ClassicCompute VMs require additional configuration for Extended Security Updates deployment, as they don't have access to the [Azure Instance Metadata Service](https://docs.microsoft.com/azure/virtual-machines/windows/instance-metadata-service), which determinines Extended Security Updates eligibility. Please contact [Microsoft support](http://support.microsoft.com/contactus?PID=17336) for more help.
```
Comment (illfated) [inline WindowsServerDocs/get-started/extended-security-updates.md:45]: Ouch. I overlooked a typo:
```suggestion
> Microsoft.ClassicCompute VMs require additional configuration for Extended Security Updates deployment, as they don't have access to the [Azure Instance Metadata Service](https://docs.microsoft.com/azure/virtual-machines/windows/instance-metadata-service), which determines Extended Security Updates eligibility. Please contact [Microsoft support](http://support.microsoft.com/contactus?PID=17336) for more help.
```

**Review by illfated — CHANGES_REQUESTED** [review]
Please add the typo correction, if you still can.

**Comment by andreiztm** [comment]
Good eyes :D I read it like 5 times and still missed it, alas.. :P

**Comment by eross-msft** [comment]
Thank you both! I'll go ahead and merge this

**Pull request merged**